client/components/summary/style.scss

```diff
@@ -216,6 +217,28 @@ $inner-border: $core-grey-light-500;
 	}
 }
 
+// IE11 doesn't auto-position grid children, so their position must be set manually
+// These arrays can be read like this:
+// When there are 9 items, the breakpoint appears after the 5th item in large screens
+// and after the 3rd one when the screen is <1365px. On screens smaller than 1100px,
+// all items are displayed one below the other.
+@include set-grid-positions( ( 1, 2, 3, 4, 5, 6, 4, 4, 5, 5 ), 'woocommerce-summary__item-container' );
```

Comment:
It's cool that this is possible, but this seems a bit too clever to be understandable. What about removing this mixin, and writing out the `has-X-items` by hand? Basically [mirroring what I've done here](https://github.com/woocommerce/wc-admin/blob/master/client/components/summary/style.scss#L106-L160), so you could do…

```scss
.has-9-items .woocommerce-summary__item-container {
	@include set-item-grid-position( 5, 9 );
	
	@include breakpoint( '<1365px' ) {
		@include set-item-grid-position( 3, 9 );
	}
}
```

Which would also let you stop passing around class names, you could have `&:nth-child(#{$i})` in `set-item-grid-position`. I know this is a bit more annoying to write out, but it's easier to understand when scanning the stylesheet.

Reply:
Right, it will make SCSS code a bit longer but easier to read. Will fix that one too! :)